Sources/Swift/Tools/UIRedactBuilder.swift

```diff
@@ -272,8 +270,8 @@ class UIRedactBuilder {
             /// The beginning will be added after all the subviews have been mapped.
             redacting.append(RedactRegion(size: layer.bounds.size, transform: newTransform, type: .clipEnd))
         }
-        for subview in view.subviews.sorted(by: { $0.layer.zPosition < $1.layer.zPosition }) {
-            mapRedactRegion(fromView: subview, relativeTo: layer, redacting: &redacting, rootFrame: rootFrame, transform: newTransform, forceRedact: enforceRedact)
+        for subLayer in (layer.sublayers ?? []).sorted(by: { $0.zPosition < $1.zPosition }) {
+            mapRedactRegion(fromLayer: subLayer, relativeTo: layer, redacting: &redacting, rootFrame: rootFrame, transform: newTransform, forceRedact: enforceRedact)
         }
```

Comment:
`m`: What stopped you from writing a test for this in `UIRedactBuilderTests`?

Reply:
There is nothing different to test.
Instead of iterating over view.subviews we're iterating over the view layer which is a lower level api, but navigating the same hierarchy pretty much.

But all the UIRedactBuilder tests in all the ways we could use it are there.

Reply:
There is nothing different to test.
Instead of iterating over view.subviews we're iterating over the view layer which is a lower level api, but navigating the same hierarchy pretty much.

But all the UIRedactBuilder tests in all the ways we could use it are there.
What do you have in mind?

Reply:
A test that fails with the old code that validates that masking for fast animation works. I have no clue how to do this 😅. If you say it's not worth the effort, merge the PR.